### DISCUSSION TITLE
[Feature] Tags to Module

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#13921] [Wishlist]
### Describe the solution you'd like
Similar to VTiger Tags management, I would like to tag record and see only those records in each module where tag is set

![image](https://user-images.githubusercontent.com/12723595/102078736-6ee8c400-3e31-11eb-8875-d3f33657205b.png)

### Is your feature request related to a problem? Please describe.
It is helpful to categories records for each entity module, and same record can be in multiple records

### Describe alternatives you've considered
No I have not tried any alternative

### Additional context
May be any other alternative is available but not able to figure out.

Regards
Jeevan Pendhari

**Comment by apcloic** [comment]
@JEEVAN28 Yes, I will also be glad to see this kind of tag feature in YF.
However, it looks like [freetag class](https://github.com/freetag/freetag) used by Vtiger is not maintained anymore.

**Comment by simple-AMS** [comment]
This would be great!

**Comment by randysrs** [comment]
Any update on Tags ? moving from Vtiger to Yeticrm .
